lib/github_notification_monitor.rb

```diff
@@ -9,6 +9,7 @@ class GithubNotificationMonitor
     "add_label"       => :add_labels,
     "remove_label"    => :remove_labels,
     "rm_label"        => :remove_labels,
+    "run-tests"       => :run_tests,
```

Comment:
Make the left hand side singular...Doing so will allow `run_test`, `run_tests`, `run-test`, `run-tests`